ui/app/templates/components/masked-input.hbs

```diff
@@ -1,12 +1,12 @@
 <div class="masked-input {{if shouldObscure "masked"}} {{if displayOnly "display-only"}} {{if allowCopy "allow-copy"}}" data-test-masked-input>
 	{{#if displayOnly}}
 		<pre class="masked-value display-only is-word-break">{{displayValue}}</pre>
-	{{else}}
+  {{else}}
   <textarea
     class="input masked-value"
     rows=1
     wrap="off"
-    placeholder="value"
+    placeholder={{value}}
```

Comment:
Having this as `{{value}}` makes it so that if you enter a value and then remove it, you still see the text as the placeholder instead. Then if you blur and focus again it is gone (because there isn't a value then).

Do we want to just leave the placeholder off? Or maybe make it so it only uses one if you specify one?

Reply:
i like the idea of only using a placeholder if we pass one in. it looks like every other instance of `<MaskedInput>` has a placeholder of `value`, so i set that as the default. i don't think it makes sense to show a placeholder (or at least not the word `value`) when typing in a password so i set that as an empty string. thanks for the suggestions!